app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_align.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import DashboardValue from "../_dashboard_value.jsx"
+
+function DashboardValueAlign() {
```

Comment:
`function DashboardValueAlign() {`   can also be written  `const DashboardValueAlign = () => {`  .... Wasn't sure if we had a standard for this yet. Carlos pushed me toward using the latter syntax. I'm actually going through all my hometour syntax now and fixing these to use the "const" syntax. What do ya'll think?

Reply:
I am referencing the syntax of the other react kits. I will leave this decision to @thestephenmarshall and I will follow!

Reply:
It isn't super critical, but yeh I would generally defer to using the lambda version.